clustered/server/src/main/java/org/ehcache/clustered/server/state/ClientMessageTracker.java

```diff
@@ -73,4 +91,8 @@ public boolean isConfigureApplicable(UUID clientId, long timestamp) {
     return true;
   }
 
+  public void reconcileTrackedClients(Set<UUID> trackedClients) {
+    messageTrackers.entrySet().removeIf(x -> !trackedClients.contains(x));
```

Comment:
This does not work - using incompatible types.
And thus also indicates there is missing testing ...